authentication/views_test.py

```diff
@@ -272,14 +273,15 @@ def register_email_exists(self, recaptcha_enabled):
     def register_email_not_exists_with_recaptcha_invalid(self):
         """Yield a function for this step"""
         self.flow_started = True
-        with patch(
-            "authentication.views.requests.post",
-            return_value=MockResponse(
-                content='{"success": false, "error-codes": ["bad-request"]}',
-                status_code=status.HTTP_200_OK,
-            ),
-        ) as mock_recaptcha_failure, override_settings(
-            **{"RECAPTCHA_SITE_KEY": "fakse"}
+        with (
+            patch(
+                "authentication.views.requests.post",
+                return_value=MockResponse(
+                    content='{"success": false, "error-codes": ["bad-request"]}',
+                    status_code=status.HTTP_200_OK,
+                ),
+            ) as mock_recaptcha_failure,
+            override_settings(**{"RECAPTCHA_SITE_KEY": "fakse"}),
```

Comment:
Doesn't matter much but we should use `fake` instead of `fakse`. This confuses the reader between a boolean and a string. e.g. Looking at this I get confused about whether it was meant to be `fake` or `false`.